### PR TITLE
data-quality-report: Check lists have enought items

### PR DESCRIPTION
**Pull request by ghost**
(And an unrelated bug fix)

### CODE REVIEW

File: indigo/dataqualityreport.py [hunk 1]
```diff
@@ -68,6 +68,9 @@ def _process(self):
             ):
                 errors.append(ValueNotANumberDataError(error))
 
+            elif error.validator == "minItems":
+                errors.append(ArrayHasToFewItemsError(error))
```
Comment (Lathrisk) [inline indigo/dataqualityreport.py]: I think this should be `too` not `to`

File: indigo/dataqualityreport.py [hunk 2]
```diff
@@ -172,3 +175,21 @@ def get_value(self):
 
     def get_priority(self):
         return 0
+
+
+class ArrayHasToFewItemsError(_DataError):
```
Comment (Lathrisk) [inline indigo/dataqualityreport.py]: As above (too vs to)

File: indigo/dataqualityreport.py [hunk 3]
```diff
@@ -172,3 +175,21 @@ def get_value(self):
...
+    def __init__(self, error):
+        self._path = "/".join([str(i) for i in error.path])
+        self._value = error.instance
+
+    def get_type(self):
+        return "array_has_to_few_items"
```
Comment (Lathrisk) [inline indigo/dataqualityreport.py]: As above (too vs to)

File: indigo/jsonschema/cached_information/fund.json.compiled_jsonschema.json [hunk 4]
```diff
@@ -228,7 +228,7 @@
                                     "type": "object"
                                 },
                                 "amount_usd": {
-                                    "description": "Actual size of funding available from the Outcome Fund (to be made for outcome payments directly or assistance to support the development of outcome-contracts)  (Conversion by INDIGO using World Bank exchange rate of the moment of expression of interest. If there is no data on dates of expression of interest, then we will use the exchange rate of the launch date)",
+                                    "description": "Actual size of funding available from the Outcome Fund (to be made for outcome payments directly or assistance to support the development of outcome-contracts)in USD. (Conversion by INDIGO using World Bank exchange rate of the moment of expression of interest. If there is no data on dates of expression of interest, then we will use the exchange rate of the launch date)",
```
Comment (Lathrisk) [inline indigo/jsonschema/cached_information/fund.json.compiled_jsonschema.json:231]: requires a space -  `)in` vs `) in`

File: indigo/jsonschema/cached_information/fund.json.compiled_jsonschema.json [hunk 5]
```diff
@@ -256,7 +256,7 @@
                                     "type": "object"
                                 },
                                 "amount_usd": {
-                                    "description": "Ambition for maximum funding available from the Outcome Fund (to be made for outcome payments directly or assistance to support the development of outcome-contracts) (Conversion by INDIGO using World Bank exchange rate of the moment of expression of interest. If there is no data on dates of expression of interest, then we will use the exchange rate of the launch date)",
+                                    "description": "Ambition for maximum funding available from the Outcome Fund (to be made for outcome payments directly or assistance to support the development of outcome-contracts)in USD. (Conversion by INDIGO using World Bank exchange rate of the moment of expression of interest. If there is no data on dates of expression of interest, then we will use the exchange rate of the launch date)",
```
Comment (Lathrisk) [inline indigo/jsonschema/cached_information/fund.json.compiled_jsonschema.json:259]: requires a space - )in vs ) in

File: indigo/templates/indigo/admin/project/data_quality_report.html [hunk 6]
```diff
@@ -64,6 +64,12 @@ <h3>Priority {{ priority }}</h3>
                         <td>{{ error.get_path }}</td>
                         <td></td>
                     </tr>
+                {% elif error.get_type == 'array_has_to_few_items' %}
```
Comment (Lathrisk) [inline indigo/templates/indigo/admin/project/data_quality_report.html]: too vs to

**Review by Lathrisk — DISMISSED** [review]
A few comments, ~~mostly~~ just typos. Some of these (the spaces) were in `data-standard/schema/fund.json` too

**Comment by ghost** [comment]
Fixed here and in other repo, as applicable

**Review by Lathrisk — APPROVED** [review]
I think you missed a couple of the comments about spaces, but they are quite minor so depends what you want to do with that :)

**Comment by ghost** [comment]
They are done in other repository and will show up here next time it's updated. The descriptions still are not used for anything here.

**Pull request merged**